tests/e2e-pw/specs/merchant/merchant-orders-full-refund.spec.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * External dependencies
+ */
+import { test, expect, Page } from '@playwright/test';
+
+/**
+ * Internal dependencies
+ */
+import { getMerchant, getShopper } from '../../utils/helpers';
+import { deactivateMulticurrency } from '../../utils/merchant';
+import * as shopper from '../../utils/shopper';
+import { goToOrder, goToPaymentDetails } from '../../utils/merchant-navigation';
+
+test.describe( 'WooCommerce Payments - Full Refund', () => {
+	let merchantPage: Page;
+	let shopperPage: Page;
+	let orderId: string;
+	let orderAmount: string;
+
+	test.beforeAll( async ( { browser } ) => {
+		merchantPage = ( await getMerchant( browser ) ).merchantPage;
+		shopperPage = ( await getShopper( browser ) ).shopperPage;
+
+		// Disable multi-currency in the merchant settings. This step is important because local environment setups
+		// might have multi-currency enabled. We need to ensure a consistent environment for the test.
+		await deactivateMulticurrency( merchantPage );
```

Comment:
Do you think it would be a good idea to check if multi-currency is enabled before proceeding? I’m mentioning this because the test fails when multi-currency is already disabled.

Alternatively, we could skip this step if the [save button is disabled](https://github.com/Automattic/woocommerce-payments/blob/dev/9959-pw-admin-disputes-view-spec/tests/e2e/utils/flows.js#L701). I believe the Playwright version of those helpers behaves a bit differently and doesn’t include that extra check for the save button.

Reply:
Thanks for noticing it, I initially missed the issue while dealing with the support phone thing. I think it's better to check if it's active instead of skipping the save button when it's disabled to avoid missing any 0981fae